setup.cfg

```diff
@@ -0,0 +1,58 @@
+[metadata]
+name = QtPy
+version = attr: qtpy.__version__
+description = Provides an abstraction layer on top of the various Qt bindings (PyQt5/6 and PySide2/6) and additional custom QWidgets.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/spyder-ide/qtpy
+author = Colin Duquesnoy and the Spyder Development Team
+author_email = spyder.python@gmail.com
+maintainer = Spyder Development Team and QtPy Contributors
+maintainer_email = spyder.python@gmail.com
+license = MIT
+license_file = LICENSE.txt
+license_files =
+    AUTHORS.md
+    LICENSE.txt
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Environment :: MacOS X
+    Environment :: Win32 (MS Windows)
+    Environment :: X11 Applications :: Qt
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: User Interfaces
+    Topic :: Software Development :: Widget Sets
+keywords = qt PyQt5 PyQt6 PySide2 PySide6
+project_urls =
+    Github = https://github.com/spyder-ide/qtpy
+    Bug Tracker = https://github.com/spyder-ide/qtpy/issues
+    Parent Project = https://www.spyder-ide.org/
+
+[options]
+packages = find:
+install_requires =
+    packaging
```

Comment:
Should this have the constraint `>=20.0` and maybe be removed for the `extras_require::test`?

Reply:
Good question; in this case the constraint is a soft one, and only has potential effect on its use in development/packaging/testing to be fully certain of consistent results, rather than something that matters for the very narrowly-scoped usage at runtime. But its not critical either way, it can be removed under the `extra` if you want.

Reply:
Then please go ahead and remove it from the `extra` section

Reply:
Sure, done.